backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -214,6 +214,9 @@ CREATE TABLE transactions(
         JSONB
 );
 
+-- Important for quickly filtering transactions in a given block.
+CREATE INDEX transactions_block_idx ON transactions (block_height);
```

Comment:
We also filter on the `index` of the transaction, so we need it to be part of the index.
```suggestion
CREATE INDEX transactions_block_idx ON transactions (block_height, index);
```